src/components/ServiceDetails/AboutService/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import { withStyles } from "@material-ui/styles";
+import { connect } from "react-redux";
+
+import { useStyles } from "./styles";
+import DemoToggler from "./DemoToggler";
+import Overview from "./Overview";
+
+const AboutService = ({ classes, isLoggedIn, service }) => {
+  return (
+    <Grid container spacing={24} className={classes.aboutContainer}>
+      <Grid item xs={12} sm={12} md={8} lg={8} className={classes.leftSideSection}>
+        <Overview description={service.description} service_url={service.url} tags={service.tags} />
+
+        <DemoToggler showDemo={isLoggedIn} classes={classes} service={service} />
+
+        <div className={classes.backToLink}>
+          <i className="fas fa-arrow-left"></i>
```

Comment:
Could be replaced with `Icon` component.

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/13/commits/1a1efe93edd5265a7a98505975a3b692232700b2)